src/lib/seo.ts

```diff
@@ -32,11 +33,29 @@ const buildManifestSEO = async (manifest: Manifest, path: string) => {
   };
 };
 
+const buildContentSEO = (frontMatter: FrontMatterContentItem, path: string) => {
```

Comment:
Adds a helper that understands our frontMatter and outputs titles and descriptions.